enterprise-metrics/Makefile

```diff
@@ -22,6 +25,25 @@ test: ## Run library tests.
 test: test/vendor
 	jsonnet -J test/lib -J test/vendor -J vendor main_test.jsonnet
 
+.PHONY: eval
+eval: ## Evaluate a minimal configuration.
+eval: test/vendor vendor
+	jsonnet -J test/lib -J test/vendor -J vendor docs.jsonnet
+
+environments/minio/jsonnetfile.json: ## Update the environment jsonnetfile.
+environments/minio/jsonnetfile.json: environments/minio/jsonnetfile.jsonnet jsonnetfile.json test/jsonnetfile.json
+	jsonnet $< > $@
+
+environments/minio/vendor: ## Vendor environment dependencies.
+environments/minio/vendor: environments/minio/jsonnetfile.json
+	cd $(@D)
+	jb install
+
+e2e-test: ## Deploy GEM to a local k3d cluster.
+e2e-test: environments/minio/vendor
```

Comment:
I would not call this an e2e test, rather a local test setup.
From an e2e test I would expect setup, test case and automatic teardown.

Reply:
Also a good point, updated in https://github.com/grafana/jsonnet-libs/pull/598/commits/f4e41500cd5736eb21b5887cf4c57af5b97c464e